tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailboxRepository.scala

```diff
@@ -75,7 +75,9 @@ object TeamMailboxRepositoryImpl {
       Right.WriteSeenFlag,
       Right.DeleteMessages,
       Right.Insert,
-      Right.Write)
+      Right.Write,
+      Right.CreateMailbox,
+      Right.PerformExpunge)
```

Comment:
We need `PerformExpunge` rights to move messages, especially when a member creates a new "custom" folder.

Reply:
If we grant CreateMailbox right so should we grant DeleteMailbox right?